index.html

```diff
@@ -78,6 +79,45 @@ <h2 id="general">General Information</h2>
     "<a href="http://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1001745">Best Practices for Scientific Computing</a>".
   </em>
 </p>
+{% elsif page.carpentry == "dc" %}
+<p>
+  <a href="{{site.dc_site}}">Data Carpentry</a>
+  aims to help researchers get their work done
+  in less time and with less pain
+  by teaching them basic research computing skills.
+  This hands-on workshop will cover basic concepts and tools,
+  including program design, version control, data management,
+  and task automation.
+  Participants will be encouraged to help one another
+  and to apply what they have learned to their own research problems.
+</p>
+<p align="center">
+  <em>
+    For more information on what we teach and why,
+    please see our paper
+    "<a href="http://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1001745">Best Practices for Scientific Computing</a>".
+  </em>
+</p>
+{% elsif page.carpentry == "lc" %}
+<p>
+  <a href="{{site.lc_site}}">Library Carpentry</a>
```

Comment:
Library Carpentry is made by librarians, for librarians to help you:

- automate repetitive, boring, error-prone tasks
- create, maintain and analyse sustainable and reusable data
- work effectively with IT and systems colleagues
- better understand the use of software in research
- and much more…

Library Carpentry introduces you to the fundamentals of computing and provides you with a platform for further self-directed learning. For more information on what we teach and why, please see our paper [Library Carpentry: software skills training for library professionals](http://doi.org/10.18352/lq.10176).